doc/settings_rstgen.py

```diff
@@ -139,8 +150,12 @@ def _populate_rst_from_settings(rst_dir, cls, version):
         r.write(f'{"="*(len(cls_orig_name))}\n\n')
         deprecated = getattr(cls, "_deprecated_version", None)
         if deprecated:
-            release_version = "20" + cls._deprecated_version.replace(".", "R")
-            r.write(f".. deprecated:: Ansys {release_version}\n\n")
+            if float(cls._deprecated_version) < 22.2:
```

Comment:
Here if the deprecated version is less that 22.2, we provide the least version supported by pyfluent, i.e. 2022R2, as the versions before that are not supported by our FluentVersion enum,

Reply:
`float(cls._deprecated_version) < 22.2` has no reliable meaning. The attribute value is invalid. Ignore it and create a Fluent defect.

Reply:
Fixed now. We are ignoring the entry like discussed.